File: fix_imports.py
#!/usr/bin/env python3
import re
import os

ui_dir = '/workspaces/Translineweb/portal/src/ui'

for filename in os.listdir(ui_dir):
    if not filename.endswith('.tsx'):
        continue
    
    filepath = os.path.join(ui_dir, filename)
    with open(filepath, 'r') as f:
        content = f.read()
    
    original = content
    
    # Fix double prefixed @radix-ui/@radix-ui
    content = re.sub(r'@radix-ui/@radix-ui', '@radix-ui', content)
    
    # Remove version specifiers from @radix-ui imports
    content = re.sub(r'"@radix-ui/([^@"]+)@[^"]*"', r'"@radix-ui/\1"', content)
    
    # Remove version specifiers from other imports
    content = re.sub(r'"lucide-react@[^"]*"', '"lucide-react"', content)
    content = re.sub(r'"cmdk@[^"]*"', '"cmdk"', content)
    content = re.sub(r'"react-day-picker@[^"]*"', '"react-day-picker"', content)
    content = re.sub(r'"class-variance-authority@[^"]*"', '"class-variance-authority"', content)
    
    if content != original:
        with open(filepath, 'w') as f:
            f.write(content)
        print(f'Fixed: {filename}')

print('Done!')

File: portal/fix-imports.js
const fs = require('fs');
const path = require('path');
const glob = require('glob');

const uiDir = path.join(__dirname, 'src', 'ui');

glob(path.join(uiDir, '*.tsx'), (err, files) => {
  if (err) {
    console.error('Error finding files:', err);
    return;
  }

  files.forEach(file => {
    let content = fs.readFileSync(file, 'utf-8');
    
    // Fix double prefixed @radix-ui/@radix-ui
    content = content.replace(/@radix-ui\/@radix-ui/g, '@radix-ui');
    
    // Remove version specifiers from imports
    content = content.replace(/"@radix-ui\/[^@]*@[^"]*"/g, match => {
      const pkg = match.replace(/@[^"]*$/, '"');
      return pkg;
    });
    
    content = content.replace(/"lucide-react@[^"]*"/g, '"lucide-react"');
    content = content.replace(/"cmdk@[^"]*"/g, '"cmdk"');
    content = content.replace(/"react-day-picker@[^"]*"/g, '"react-day-picker"');
    content = content.replace(/"class-variance-authority@[^"]*"/g, '"class-variance-authority"');
    
    fs.writeFileSync(file, content, 'utf-8');
    console.log('Fixed:', path.basename(file));
  });

  console.log('Done!');
});
